backend/backend/backend-src/server.ts

```diff
@@ -69,14 +85,6 @@ app.get('/holdings/:address/:ticker', holdings.handleGetHoldings)
 // transaction volumes
 app.get('/txvol/:address/:ticker', txvol.handleGetTxVol)
 
-// for dev/testing and as a heartbeat
```

Comment:
This endpoint is used by App Runner to check if the server is up.  It can't be removed, but it can return something else, like `res.status(200).json({status: 'ok})`.

Reply:
I have added another API. `/status` for this purpose which is more readable. But I will change this to the above suggestion if it is already being used.